integration_tests/__tests__/__snapshots__/transform-test.js.snap

```diff
@@ -0,0 +1,27 @@
+exports[`babel-jest instruments only specific files and collects coverage 1`] = `
+"Using Jest CLI v14.0.0, jasmine2, babel-jest
+----------------------------|----------|----------|----------|----------|----------------|
+File                        |  % Stmts | % Branch |  % Funcs |  % Lines |Uncovered Lines |
+----------------------------|----------|----------|----------|----------|----------------|
+ this-directory-is-covered/ |    83.33 |      100 |       50 |    83.33 |                |
+  covered.js                |    83.33 |      100 |       50 |    83.33 |             18 |
+----------------------------|----------|----------|----------|----------|----------------|
+All files                   |    83.33 |      100 |       50 |    83.33 |                |
```

Comment:
I take it we still have those bugs where statements and functions aren't covered in full?